primitives/runtime/src/lib.rs

```diff
@@ -801,6 +801,23 @@ impl Drop for SignatureBatching {
 	}
 }
 
+/// Describes on what should happen with a transaction.
```

Comment:
Nit: a transaction is a fairly overloaded term. Prefer using a "storage transaction" to deferentiate.

```suggestion
/// Describes on what should happen with a storage transaction.
```